apps/volk_profile.cc

```diff
@@ -67,6 +67,13 @@ int main(int argc, char *argv[]) {
     profile_options.add((option_t("json", "j", "Write results to JSON file named as argument value", set_json)));
     profile_options.add((option_t("path", "p", "Specify the volk_config path", set_volk_config)));
     profile_options.parse(argc, argv);
+    
+    for (int arg_number = 0; arg_number < argc; ++arg_number) {
+        if (std::string("--help") == std::string(argv[arg_number]) ||
+            std::string("-h") == std::string(argv[arg_number])) {
```

Comment:
Albeit a simple method of checking for an argument: 

Since `option_list` handles `-h`||`--help` specially internally, I'd argue that adding a member `help` to the class that's set to `true` iff the `parse` hits that argument in https://github.com/gnuradio/volk/blob/9955b620641244c62dfed65c6de1d0b89b38b922/apps/volk_option_helpers.cc#L112 wouldn't be less beautiful than this; it would reduce your patch to this file to `if(profile_options.help) { return 0; }`.